### DISCUSSION TITLE
Can't apply selected theme

### DISCUSSION THREAD

**Discussion by vlador4ik** [ohmyzsh/ohmyzsh#12174] [Help] [answered]
Here is .zshrc file. I added to this file next code:
<img width="642" alt="image" src="https://github.com/ohmyzsh/ohmyzsh/assets/42747832/1c3763e5-5f8c-43a6-b9ff-5c642fa8168a">
After that I restarted the terminal but nothing happened ( theme didn't change ). Can anyone help me with change theme for my terminal? Thanks!

**Comment by mcornella** [comment, marked as answer]
You are editing `.szhrc` (note incorrect ordering of letters). It should be `.zshrc`.

**Reply by vlador4ik** [reply]
Ohh. Thank you!!
